vsintegration/src/vs/FsPkgs/FSharp.Project/Common.Source.CSharp/Project/HierarchyNode.cs

```diff
@@ -2327,8 +2320,13 @@ public void OnItemAdded(HierarchyNode parent, HierarchyNode child)
                 throw new ArgumentNullException("child");
             }
 
-            HierarchyNode foo;
-            foo = this.projectMgr == null ? this : this.projectMgr;
+            if (parent.onChildAdded != null)
+            {
+                HierarchyNodeEventArgs args = new HierarchyNodeEventArgs(child);
+                parent.onChildAdded(parent, args);
+            }
+
+            var foo = this.projectMgr ?? this;
```

Comment:
Since we are cleaning up this code, can we have a proper variable name rather than __foo__.  We might as well clean it up fully rather than partially.